web/src/beta/features/WorkspaceSetting/hooks.ts

```diff
@@ -52,9 +53,11 @@ export default () => {
   const handleCreateWorkspace = useCallback(
     async ({ name }: WorkspacePayload) => {
       try {
-        const { status } = await useCreateWorkspace(name);
-        if (status === "success") {
-          console.log("Workspace created successfully");
+        if (name) {
+          const { status } = await useCreateWorkspace(name);
+          if (status === "success") {
+            console.log("Workspace created successfully");
+          }
```

Comment:
We don't need this console.log here, even when we need one we should handle it on api code.